<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateColumnForm</class>
 <widget class="QDialog" name="CreateColumnForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Column</string>
  </property>
  <widget class="QComboBox" name="dataTypesComboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="createColumnButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="columnName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="columnNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Column Name</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="isPrimaryButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary Key</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="isNullableButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nullable</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
